src/api.ts

```diff
@@ -1,17 +1,24 @@
-// const BASE_URL = 'https://mate.academy/students-api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// function wait(delay: number): Promise<void> {
-//   return new Promise(resolve => {
-//     setTimeout(resolve, delay);
-//   });
-// }
+const BASE_URL = 'https://mate.academy/students-api';
 
-// function get<T>(url: string): Promise<T> {
-//   return wait(1000)
-//     .then(() => fetch(BASE_URL + url))
-//     .then(res => res.json());
-// }
+function wait(delay: number): Promise<void> {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
 
-export const getTodos = () => {};
+function get<T>(url: string): Promise<T> {
+  return wait(1000)
```

Comment:
You don't need to wait 1 sec before each request, `wait` function is given to test Loader:

<img width="857" alt="image" src="https://user-images.githubusercontent.com/52370890/181051590-b37cbc55-b43b-4056-93b9-a909ff56e439.png">

Reply:
Let's delete this wait from `get` function